CMakeLists.txt

```diff
@@ -110,6 +111,7 @@ else()
     exec_program(${ROOT_CONFIG_EXECUTABLE} ARGS "--prefix" OUTPUT_VARIABLE ROOTSYS RETURN_VALUE RETVAR)
     set(ROOTSYS ${ROOTSYS} CACHE INTERNAL "")
     set(ROOT_LIBRARIES Core RIO Net Hist Gpad Tree Rint Matrix MathCore)
+    set(ROOT_TUTORIALS_DIR ${ROOTSYS}/tutorials)
```

Comment:
Because it's in the `else()` branch of `if(DEFINED ROOT_SOURCE_DIR)`. I can also get it from `root-config` here if that's the preferred way...

(and I noticed that I messed up the Windows fix, it needs to go into the first commit)

Reply:
> I can also get it from root-config here if that's the preferred way...

Please :)

Reply:
Implemented except for Windows - the naive approach didn't work there, so I reverted back to `set(ROOT_TUTORIALS_DIR ${ROOTSYS}/tutorials)`. Might need @bellenot to take a look, but on the other hand I'm not sure if roottest works on an installed version of ROOT so maybe we only need to care about the setup used for CI builds on Windows?

Reply:
> Implemented except for Windows - the naive approach didn't work there, so I reverted back to `set(ROOT_TUTORIALS_DIR ${ROOTSYS}/tutorials)`. Might need @bellenot to take a look, but on the other hand I'm not sure if roottest works on an installed version of ROOT so maybe we only need to care about the setup used for CI builds on Windows?

Well, last time I tried it was working. And I'll check what we can get with `exec_program(${ROOT_CONFIG_EXECUTABLE} ARGS "--tutdir" OUTPUT_VARIABLE ROOT_TUTORIALS_DIR)`